src/gpu/kmeans/kmeans_h2o4gpu.cu

```diff
@@ -1,37 +1,37 @@
 /*!
```

Comment:
This file seems to be all formatting changes. I cant see if you have actually changed anything. It is better to only commit actual changes and deal with formatting issues in a separate commit or PR.

I will take another look at this if you revert the formatting changes.

Reply:
There is no change in it, I kept my code isolated from the rest of the project. And my hope is removing this file. Sorry for the mess.